bin/diagnostics.bash

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+here="$(dirname "$(readlink -f "$0")")"
+# shellcheck source=bin/common.bash
+source "${here}/common.bash"
+
+cluster="${1}"
+
+file="${CK8S_CONFIG_PATH}/diagnostics-$(date +%y%m%d%H%M%S).log"
+touch "${file}"
+
+sops_encrypt_file() {
+    if [ -z "${CK8S_PGP_FP:-}" ]; then
+        sops_encrypt "${file}"
+        return
+    fi
+
+    log_info "Encrypting ${file}"
+
+    sops --pgp "${CK8S_PGP_FP}" -e -i "${file}"
```

Comment:
:thinking: Who would this file be encrypted for? I thought it should be encrypted for the SME team, and not necessarily the platform administrator. Perhaps we need to look into a GPG [group key](https://www.gnupg.org/documentation/manuals/gnupg/GPG-Key-related-Options.html#index-group). :thinking:

Reply:
I agree. Could we then just make the script issue a warning like:

```
Encrypting the key for the fingerprint $FINGERPRINT.
If you want to send diagnostic data to Elastisys, make sure to do:

CK8S_PGP_FP=<fingerprint provided during onboarding> ./bin/ck8s diagnostics [sc|wc]

If in doubt, contact support@elastisys.com.
```

Reply:
Done, looks like this now:
![image](https://github.com/elastisys/compliantkubernetes-apps/assets/112404905/df4bfd19-44d6-4679-b839-527ee06b23c7)

Reply:
Love it! :heart: